### ISSUE TITLE
Allow to bind certain apps to open on a defined workspace

### ISSUE THREAD

**Issue by donadigo** [elementary/switchboard-plug-pantheon-shell#116]
I think it would actually encourage people to use workspaces more by letting them to configure what app opens on what workspace by default so that when they open an app, it opens automatically on a specific workspace.

Say e.g:
First workspace: browser
Second workspace: code editor, terminal
Thrid workspace: music player

It could also probably be reported in https://github.com/elementary/switchboard-plug-applications instead of here.

**Comment by danrabbit** [comment]
I'm not sure if this is necessarily possible or practical because of dynamic workspaces. What happens if you open Music while you only have 1 workspace?

**Comment by donadigo** [comment]
Yeah, this is certainly a problem here that I don't know how to approach yet. A while ago I built a gala plugin that does this and it works nicely except for that one thing.
